Authenticator/src/Authy/IDStores/LDAP.pm

```diff
@@ -114,7 +114,7 @@ sub initialize {
             RetryDelay => [$connection_retry_delay],
         );
     };
-    die "Could not initialize LDAP connection pool: $@\n";
+    die "Could not initialize LDAP connection pool: $@\n" if $@;
```

Comment:
@benrubson I noticed that now the `die` instruction may not be executed. 

Can you please explain what would be an implication of that?

Reply:
@LucasVidal thank you for your review 👍
`$@` contains the error if the `eval` block above fails.
If the block above does not fail, there's simply no reason to `die` :)
Otherwise this `initialize` function will never return.

Reply:
This is how `eval` / `die` is generally handled, and it's like this everywhere else in this module.
I think it's a simple typo here.